SU2_CFD/src/solver_structure.cpp

```diff
@@ -4825,6 +4825,365 @@ void CSolver::Read_InletFile_ASCII(CGeometry *geometry, CConfig *config, string
   
 }
 
+void CSolver::PrintNewInletData (passivedouble *Inlet_InterpolatedData, unsigned long nVertex,string Marker_Tag)
+{
+
+ofstream myfile;
+myfile.precision(16);
+myfile.open("Interpolated_Data_"+Marker_Tag+".dat",ios_base::out);
+
+
+if (myfile.is_open())
+{  
+            for (unsigned long iVertex = 0; iVertex < nVertex; iVertex++) {
+
+                  for  (unsigned long iVar=0; iVar < (maxCol_InletFile+nDim); iVar++)
+                    {
+                      myfile<<Inlet_InterpolatedData[iVertex*(maxCol_InletFile+nDim)+iVar]<<"\t";
+                    }
+                  myfile<<endl;
+                }
+        myfile.close();
+}
+  else
+    cout<<"file cannot be opened"<<endl;
+}
```

Comment:
tabs / spaces issue here? Also please try to be consistent with the rest of the code regarding where you leave the {}'s (I know it is not easy to define what consistent means, but we usually have `...) {`)

Reply:
Yes, this skipped my sight before. I have to fix it.